doc/integrator/requirements.rst

```diff
@@ -55,6 +55,17 @@ default is: ``http://localhost:8080/print-c2cgeoportal-{instanceid}/pdf/``.
 And by default the 'webapps' folder is ``/srv/tomcat/tomcat1/webapps``,
 to change it set the ``PRINT_OUTPUT`` value in the makefile.
 
+For MapFish Print to function properly it is recommended to increase the default
+Java Heap Size for Tomcat (parameter ``-Xmx`` and ``-Xms``). On servers managed
+by Camptocamp check the size in the file ``/srv/tomcat/tomcat1/bin/setenv-local.sh``:
```

Comment:
This:

> On servers managed by Camptocamp check the size in the file ``/srv/tomcat/tomcat1/bin/setenv-local.sh``:

is wrong, it should be done by the sysadmins (as you correctly ask) but the sysadmins will do it in the `/srv/tomcat/tomcat1/bin/setenv.sh` file ...